shardy/dialect/sdy/ir/ops.td

```diff
@@ -273,21 +273,6 @@ def DataFlowEdgeOp : Sdy_Op<"data_flow_edge",
 
     An op can have multiple data flow edges that are orthogonal to one another.
 
-    For example:
```

Comment:
I'm not sure about removing this example, I think it's still useful when trying to understand this op. wdyt?

Reply:
+1 like having an example here so that anyone reading the code can understand the implementation of the dataflow interface and know how they are inserted for a common op.